webui: ui-test flaky

See for example https://teamcity.cockroachdb.com/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=1585634&_focus=1458

```
38]
[Test web UI] HeadlessChrome 78.0.3904 (Linux 0.0.0): Executed 130 of 320 SUCCESS (0 secs / 3.06 secs)
[11:56:38]
[Test web UI]  [1A [2KHeadlessChrome 78.0.3904 (Linux 0.0.0) Query Management Saga integration tests AUTO_REFRESH/STOP_AUTO_REFRESH action sets inRunning flag on reducer when query is running. FAILED
[11:56:38]
[Test web UI] 	AssertionError: expected 2 to equal 1
[11:56:38]
[Test web UI] 	    at src/redux/queryManager/saga.spec.ts:20980:34
[11:56:38]
[Test web UI] HeadlessChrome 78.0.3904 (Linux 0.0.0): Executed 131 of 320 (1 FAILED) (0 secs / 3.079 secs)
[11:56:38]
[Test web UI]  [1A [2KWARN: '[WARNING]:', 'Saga exceeded async timeout of 250ms'
[11:56:38]
[Test web UI] HeadlessChrome 78.0.3904 (Linux 0.0.0): Executed 131 of 320 (1 FAILED) (0 secs / 3.079 secs)
[11:56:39]
[Test web UI]  [1A [2KWARN: '[WARNING]:', 'Saga exceeded async timeout of 250ms'
[11:56:39]
[Test web UI] HeadlessChrome 78.0.3904 (Linux 0.0.0): Executed 131 of 320 (1 FAILED) (0 secs / 3.079 secs)
```

--- a/pkg/ui/src/redux/queryManager/saga.spec.ts
+++ b/pkg/ui/src/redux/queryManager/saga.spec.ts
@@ -66,7 +66,7 @@ describe("Query Management Saga", function() {
             it("immediately runs a saga when refresh is called", function() {
                 return expectSaga(queryManagerSaga)
                     .dispatch(refresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 1);
                     });
@@ -75,7 +75,7 @@ describe("Query Management Saga", function() {
                 return expectSaga(queryManagerSaga)
                     .dispatch(refresh(testQueryCounter))
                     .dispatch(refresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 1);
                     });
@@ -85,7 +85,7 @@ describe("Query Management Saga", function() {
                     .dispatch(refresh(testQueryCounter))
                     .delay(10)
                     .dispatch(refresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 2);
                     });
@@ -94,7 +94,7 @@ describe("Query Management Saga", function() {
                 return expectSaga(queryManagerSaga)
                     .withReducer(queryManagerReducer)
                     .dispatch(refresh(testQueryError))
-                    .run()
+                    .silentRun()
                     .then(runResult => {
                         assert.isObject(runResult.storeState[testQueryError.id]);
                         assert.equal(runResult.storeState[testQueryError.id].lastError, sentinelError);
@@ -107,7 +107,7 @@ describe("Query Management Saga", function() {
                     .delay(10)
                     .dispatch(refresh(testQueryCounter))
                     .dispatch(stopAutoRefresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 2);
                     });
@@ -118,7 +118,7 @@ describe("Query Management Saga", function() {
                 return expectSaga(queryManagerSaga)
                     .dispatch(autoRefresh(testQueryCounter))
                     .dispatch(stopAutoRefresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 1);
                     });
@@ -128,7 +128,7 @@ describe("Query Management Saga", function() {
                     .dispatch(refresh(testQueryCounter))
                     .dispatch(autoRefresh(testQueryCounter))
                     .dispatch(stopAutoRefresh(testQueryCounter))
-                    .run()
+                    .silentRun()
                     .then(() => {
                         assert.equal(queryCounterCalled, 1);
                     });
@@ -141,6 +141,7 @@ describe("Query Management Saga", function() {
                 let queryCalled = 0;
                 const selfStopQuery = {
                     id: "selfStopQuery",
+                    refreshInterval: moment.duration(50),
                     querySaga: function* (): IterableIterator<void> {
                         queryCalled++;
                         if (queryCalled > 3) {
@@ -148,11 +149,11 @@ describe("Query Management Saga", function() {
                         }
                     },
                 };
-                tester
-                    .dispatch(autoRefresh(testQueryCounter))
-                    .dispatch(autoRefresh(testQueryCounter))
-                    .dispatch(autoRefresh(testQueryCounter))
-                    .run()
+                return tester
+                    .dispatch(autoRefresh(selfStopQuery))
+                    .dispatch(autoRefresh(selfStopQuery))
+                    .dispatch(autoRefresh(selfStopQuery))
+                    .silentRun(250)
                     .then(() => {
                         assert.equal(queryCalled, 5);
                     });
@@ -160,7 +161,7 @@ describe("Query Management Saga", function() {
             it("Uses retry delay when errors are encountered", function() {
                 return expectSaga(queryManagerSaga)
                     .dispatch(autoRefresh(testQueryError))
-                    .run({timeout: 200})
+                    .silentRun(200)
                     .then(() => {
                         // RefreshTimeout is high enough that it would only be
                         // called once.
@@ -179,7 +180,7 @@ describe("Query Management Saga", function() {
                     .withReducer(queryManagerReducer)
                     .dispatch(refresh(neverResolveQuery))
                     .dispatch(refresh(testQueryCounter))
-                    .run({timeout: 10})
+                    .silentRun()
                     .then(runResult => {
                         assert.isTrue(runResult.storeState[neverResolveQuery.id].isRunning);
                         assert.isFalse(runResult.storeState[testQueryCounter.id].isRunning);
@@ -203,7 +204,7 @@ describe("Query Management Saga", function() {
                     const tester = expectSaga(queryManagerSaga)
                         .dispatch(refresh(explicitResolveQuery));
 
-                    const testFinished = tester.run();
+                    const testFinished = tester.silentRun();
                     await delay(0);
 
                     // Query is now in progress, waiting on explicit resolve to

